tests/staticfiles_tests/test_handlers.py

```diff
@@ -0,0 +1,15 @@
+from django.contrib.staticfiles.handlers import ASGIStaticFilesHandler
+from django.core.handlers.asgi import ASGIHandler
+from django.test import AsyncRequestFactory, SimpleTestCase, modify_settings
+
+
+class TestASGIStaticFilesHandler(SimpleTestCase):
+    async_request_factory = AsyncRequestFactory()
+
+    @modify_settings(INSTALLED_APPS={'append': 'staticfiles_tests.apps.test'})
+    async def test_get_async_response(self):
+        request = self.async_request_factory.get('/static/test/file.txt')
+        handler = ASGIStaticFilesHandler(ASGIHandler())
+        response = await handler.get_response_async(request)
+        response.close()
```

Comment:
Just a note: without `response.close()` here, the test ends up with the following:

```
test_get_async_response (tests.staticfiles_tests.test_handlers.TestASGIStaticFilesHandler) ... ok

----------------------------------------------------------------------
Ran 1 test in 0.005s

OK
Exception ignored in: <_io.FileIO name='/Users/joshmassover/Development/django/tests/staticfiles_tests/apps/test/static/test/file.txt' mode='rb' closefd=True>
ResourceWarning: unclosed file <_io.FileIO name='/Users/joshmassover/Development/django/tests/staticfiles_tests/apps/test/static/test/file.txt' mode='rb' closefd=True>
```